src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportTaskMetricsNotification.cs

```diff
@@ -0,0 +1,49 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Fhir.Core.Features.Metrics;
+
+namespace Microsoft.Health.Fhir.Core.Features.Operations.Import
+{
+    public class ImportTaskMetricsNotification : IMetricsNotification
+    {
+        public ImportTaskMetricsNotification(
+            string id,
+            string status,
+            DateTimeOffset createdTime,
+            DateTimeOffset endTime,
+            long? dataSize,
+            long? succeedCount,
+            long? failedCount)
+        {
+            Id = id;
+            Status = status;
+            CreatedTime = createdTime;
+            EndTime = endTime;
+            DataSize = dataSize;
+            SucceedCount = succeedCount;
+            FailedCount = failedCount;
+        }
+
+        public string FhirOperation { get; }
```

Comment:
>FhirOperation [](http://example.com/codeflow?start=22&length=13)

Should we set it to `Import`?

Reply:
yes, it should be set to import ,already set it in constructor.